frontend/javascripts/dashboard/folders/details_sidebar.tsx

```diff
@@ -282,7 +287,7 @@ function FolderDetails({
           <div style={{ marginBottom: 4 }}>
             <FolderTeamTags folder={folder} />
           </div>
-          <MetadataTable datasetOrFolder={folder} />
+          <MetadataTable datasetOrFolder={folder} key={`${folder.id}#folder`} />
```

Comment:
please add a comment that the key is crucial to enforce remounts when the id changes.

Reply:
I also added a comment to the component itself